style.css

```diff
@@ -0,0 +1,522 @@
+}
+
+
+/*** SIGN UP FORM ****/
+
+
+form{
+  box-sizing: border-box;
+  width:50%;
+}
+
+.field-container{
+  margin-top:5px;
+  width:70%;
+}
+
+.the-form{
+  box-sizing: border-box;
+  background-color: #ecded5;
+  display: flex;
+  justify-content: center;
+  margin: 60px 0px 0px 0px;
+}
+
+.p-form-main{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+  color: black;
+  text-align: justify;
+}
+
+.p-form{
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.1em;
+  color: black;
+  font-weight: 900;
+  text-align: left;
+  margin: 15px 5px;
+}
+
+.p-form-small{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+  font-style: bold;
+}
+
+.amount-container{
+  display:inline-flex;
+}
+
+.amount{
+  box-sizing: border-box;
+  margin: 5px;
+}
+
+/* radio buttons styling */
+
+.radio-button-container{
+  display: block;
+  position: relative;
+  padding-left: 25px;
+  margin: 10px 5px;
+  font-size: 16px;
+  user-select: none;
+}
+
+
+.radio-button-container input{
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 16px;
+  width: 16px;
+  transform: translate(-180%, -255%);
+  z-index: 1;
+}
+
+.custom-radio{
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 17px;
+  width: 17px;
+  background-color: #eee;
+  cursor: pointer;
+  border-radius: 50%;
+}
+
+.radio-button-container:hover input ~ .custom-radio{
+  background-color: #444444;
+}
+
+.radio-button-container input:checked ~ .custom-radio{
+  background-color: #444444
+} 
+
+.person{
+  box-sizing: border-box;
+  margin: 15px 5px;
+}
+
+.news{
+  display: flex;
+  flex-direction: column;
+  margin: 0px 5px;
+}
+
+/* check box styling */
+
+.checkbox-container{
+  display: block;
+  position: relative;
+  padding-left: 25px;
+  margin: 10px 5px;
+  font-size: 16px;
+  user-select:none;
+}
+
+.checkbox-container input{
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 17;
+  width: 17;
+  transform: translate(-210%, -740%);
+  z-index: 1;
+}
+
+.custom-checkbox{
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 17px;
+  width: 17px;
+  background-color: #eee;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+.checkbox-container:hover input ~ .custom-checkbox{
+  background-color: #444444;
+}
+
+.checkbox-container input:checked ~ .custom-checkbox{
+  background-color: #444444
+}
+
+.custom-checkbox:after {
+  content: "";
+  position: absolute;
+  width:  3px;
+  height: 6px;
+  border: solid white;
+  border-width: 0 3px 3px 0;
+  top: 5px;
+  left: 7px;
+  transform: rotate(45deg);
+}
+
+
+.bank{
+  box-sizing: border-box;
+  margin: 10px 5px;
+}
+
+.submit-form{
+  margin: 15px 5px;
+}
+
+input[type = "text"]{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.8em;
+}
+
+input[type = "number"]{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.8em;
+  height: 100px;
+}
+
+input[type = "submit"]{
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.8em;
+}
```

Comment:
When you have the same styling for two elements, you could just separate the elements with a comma:
```
input[type="text"], input[type="submit"] {
  font-family: 'Montserrat', sans-serif;
  font-size: 0.8em;
}
```